compiler-core/src/parse/error.rs

```diff
@@ -358,6 +388,11 @@ pub enum ParseErrorType {
         field_type: Option<TypeAst>,
     },
     CallInClauseGuard, // case x { _ if f() -> 1 }
+    InvalidImportSyntax {
```

Comment:
I'm not sure what this means. Can you add documentation and a clearer name please.